### PR TITLE
update IN 2023 tax rules

### LINKED ISSUE #4175: Add 2023 Indiana Tax Rules
2023 Income Tax Form Instructions: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
(All tax parameters haven't been updated except for Indiana tax rate for 2023)

### PR DESCRIPTION
**Pull request by yuantian0204**
Fixes #4175

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - 2023 IN tax rules and refs
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - 2023 Indiana income tax updates.
```

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
 description: Indiana provides this base exemption amount.
 values:
-  2021-01-01: 1_000
+  2021-01-01: 1_000 #married filing jointly should be $2_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml:3]: remove comment but do we account for this in the code

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml [hunk 3]
```diff
@@ -1,10 +1,12 @@
 description: Indiana provides this additional exemption amount for qualifying dependent children.
 values:
-  2021-01-01: 1_500
+  2021-01-01: 1_500 #first qualified year is 3_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml:3]: remove comment, do we account for this in the code?

File: policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/decoupled.yaml [hunk 4]
```diff
@@ -5,6 +5,13 @@ metadata:
...
       href: https://iga.in.gov/laws/2021/ic/titles/6#6-3.1-21
+    - title: 2023 IT-40 Tax Form Instructions Booklet, pg.3
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+    - title: 2023 IT-40 Income Tax Form, Schedule IN-EIC, pg. 2
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+
 values:
   2010-01-01: false
   2011-01-01: true
+  2023-01-01: false
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/decoupled.yaml:16]: add a 2023 test to the variable that is affected

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml (hunk 3 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml:3]: We would add 
2023-01-01: 3_000

Add a birth year parameter and then check if the chidl is born after 2023 which then allows for a higher amount

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
 metadata:
   unit: currency-USD
   period: year
   label: Indiana exemptions additional amount for dependent childen
   reference:
     - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(5)(A)
       href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-1-3.5
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml:10]: could you find the legal section and see how the law shows this value

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml (hunk 3 above)
Comment (yuantian0204) [reply policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml:3]: It looks like we didn't account for this in the code. I added a new parameter for this value but wasn't sure about how to modify the code, maybe we can talk about it on Monday

File: policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/decoupled.yaml (hunk 4 above)
Comment (yuantian0204) [reply policyengine_us/parameters/gov/states/in/tax/income/credits/earned_income/decoupled.yaml:16]: I still can't run the test due to the problem I had a couple of weeks before; I did check the test section and it's this screenshot
Screenshot 2024-07-21 at 3 44 25 PM
I think we need to modify the outdated parameters (0.09 and it should be 0.10 now)

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml (hunk 2 above)
Comment (yuantian0204) [reply policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml:3]: We didn't account for this in the code but I modified this to add a breakdown by filing status

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml [hunk 6]
```diff
@@ -8,3 +8,19 @@ metadata:
...
     - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(3)-(4)
       href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-1-3.5
+    - title: 2023 IT-40 Income Tax Form Instruction Booklet, pg. 30
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+    - title: 2024 IC 6-3-1-3.5 "Adjusted gross income" (a)(3)
+      href: https://iga.in.gov/laws/2024/ic/titles/6#6-3-1-3.5
+  breakdown: 
+    - filing_status
+JOINT:
+2021-01-01: 2_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml:18]: indent

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml [hunk 7]
```diff
@@ -8,3 +8,19 @@ metadata:
...
+SURVIVING_SPOUSE:
+2021-01-01: 1_000
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/in/tax/income/exemptions/base/amount.yaml:20]: and others

File: policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/birthyear_threshold_amount.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+  unit: currency-USD
+  period: year
+  label: Indiana exemptions additional amount for dependent childen
+  reference: 
+    - title: 2023 IT-40 Income Tax Form Instruction Booklet, pg. 26
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
+    - title: 2024 IC 6-3-1-3.5 "Adjusted gross income" (a)(5)(A)
+      href: https://iga.in.gov/laws/2024/ic/titles/6#6-3-1-3.5
+value:
+  2023-01-01: 3000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/exemptions/additional/birthyear_threshold_amount.yaml:12]:
```suggestion
  2023-01-01: 3_000
```

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**